From 1bc3943ac6f01e481640751e43f96cdff8557914 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Fri, 30 Nov 2018 01:27:13 +0300
Subject: [PATCH 10/11] arm: dts: ulcb: Set all RAVB pins strength to maximum

This sets maximum strength for all RAVB pin on H3ULCB-HAD.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 arch/arm/dts/ulcb.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/ulcb.dtsi b/arch/arm/dts/ulcb.dtsi
index 17a0578..c561d1d 100644
--- a/arch/arm/dts/ulcb.dtsi
+++ b/arch/arm/dts/ulcb.dtsi
@@ -276,7 +276,7 @@
 		pins_mii_tx {
 			pins = "PIN_AVB_TX_CTL", "PIN_AVB_TXC", "PIN_AVB_TD0",
 			       "PIN_AVB_TD1", "PIN_AVB_TD2", "PIN_AVB_TD3";
-			drive-strength = <12>;
+			drive-strength = <24>;
 		};
 	};
 
-- 
2.7.4

